src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,29 @@
+package seedu.addressbook.data.tag;
+
+import seedu.addressbook.data.person.Person;
+import seedu.addressbook.data.tag.Tag;
+
+/*
+ * Represents the association between Person and Tag
+ */
+public class Tagging {
+    private String action;
```

Comment:
I can't agree that this is an Association Class. Why? you are not maintaining the Person and Tag associated with the tagging. 
Hints: Maintain the Person and the Tag class in the Tagging class cited to Strings name, tag

It is not a good idea to use String type to maintain the "action". Consider a boolean or an enum.

Where are you maintain the list of tagging?